TELEMETRY.md

```diff
@@ -21,12 +21,12 @@ No properties for event
 
 [src/interactive-window/commands/commandRegistry.ts](https://github.com/microsoft/vscode-jupyter/tree/main/src/interactive-window/commands/commandRegistry.ts)
 ```typescript
-        }
-    }
-
-    @captureTelemetry(Telemetry.AddCellBelow)
-    private async addCellBelow(): Promise<void> {
-        await this.getCurrentCodeWatcher()?.addEmptyCellToBottom();
+        }

```

Comment:
Why did this change so much? Looks like whitespace? Did you accidentally format it?

Reply:
just the pre-commit auto generation that Don added

Reply:
Yeah but why did it reformat it? Seems weird.